WcaOnRails/app/models/registration.rb

```diff
@@ -192,4 +192,24 @@ def events_to_associated_events(events)
       registration_competition_events.find_by_competition_event_id(competition_event.id) || registration_competition_events.build(competition_event: competition_event)
     end
   end
+
+  def serializable_hash(options = {})
+    json = {
+      class: self.class.to_s.downcase,
+      id: id,
+      competition_id: competition_id,
+      user_id: user_id,
+      event_ids: events.map { |e| e.id },
+      accepted_at: accepted_at,
```

Comment:
Technically both `accepted_at` and `deleted_at` are not visible.
Should I replace this with a `status` field, similarly to what we have in the csv?